src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -27,5 +27,7 @@
     public static final Prefix PREFIX_START_DATETIME = new Prefix("s/");
     public static final Prefix PREFIX_END_DATETIME = new Prefix("e/");
     public static final Prefix PREFIX_STOP_REPEAT = new Prefix("until/");
+    public static final Prefix PREFIX_INDEX = new Prefix("i/");
```

Comment:
I don't think there's a need for this. There seems to be some inbuilt Index parsing already. Refer to `EditCommandParser.java`:
```
try {
    index = ParserUtil.parseIndex(argMultimap.getPreamble());
} catch (ParseException pe) {
    throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
}
```

Yup so there's a special class for `Index` already, do try to use it.

Reply:
Resolved in b9dde84.